doc/related_projects.rst

```diff
@@ -231,6 +231,15 @@ and tasks.
   Feature-engine allows the application of preprocessing steps to selected groups
   of variables and it is fully compatible with the Scikit-learn Pipeline.
 
+**Topological Data Analysis**
+
+- `giotto-tda <https://github.com/giotto-ai/giotto-tda>`_ A library for `Topological Data Analysis
```

Comment:
I know we are not consistent with `rst` file, but can this be updated to use < 80 characters per line?

Reply:
Thanks for the review! This is now done.